### PR TITLE
Attempt at addressing some feedback from Aaron 

### PR DESCRIPTION
**Pull request by b---c**
https://mailarchive.ietf.org/arch/msg/oauth/v57qYaLOuywWCNhzBIJFgtutBfI/ 

@aaronpk

### CODE REVIEW

File: main.md [hunk 1]
```diff
@@ -796,7 +796,7 @@ In this case, the AS added the following example claims to the JWT-based access
 
 ## Token Introspection {#token_introspection}
 
-Token introspection [@!RFC7662] provides a means for an RS to query the AS to determine information about an access token. The token introspection response provides the RS with the authorization details applicable to it as a top-level JSON element along with the claims the RS requires for request processing. The `authorization_details` member contains the same structure defined in (#authz_details), potentially filtered and extended for the RS making the introspection request.
+Token introspection [@!RFC7662] provides a means for an RS to query the AS to determine information about an access token. It is RECOMMENDED that the applicable authorization detail information be conveyed with `authorization_details` as a top-level member of the introspection response JSON. The `authorization_details` member MUST contain the same structure defined in (#authz_details), potentially filtered and extended for the RS making the introspection request.
```
Comment (adeinega) [inline main.md:799]: As a side remark, https://datatracker.ietf.org/doc/html/draft-ietf-oauth-jwt-introspection-response-12 comes along with RFC7662, and in its case, `authorization_details` won't be a top-level member of the introspection response.

Comment (bc-pi) [reply main.md:799]: It's a top level member of the token_introspection claim in that case, which is the token introspection response.

Comment (jricher) [inline main.md:799]: I think instead we want to say that if you're going to do it, you have to do it exactly this way:

```suggestion
Token introspection [@!RFC7662] provides a means for an RS to query the AS to determine information about an access token. If the AS includes authorization details for the token in its response, the applicable authorization detail information MUST be conveyed with `authorization_details` as a top-level member of the introspection response JSON object. The `authorization_details` member MUST contain the same structure defined in (#authz_details), potentially filtered and extended for the RS making the introspection request.
```

File: main.md [hunk 2]
```diff
@@ -950,11 +950,11 @@ The Security Considerations of [@RFC6749], [@RFC7662], and [@RFC8414] also apply
 
 # Privacy Considerations {#privacy_considerations}
 
-Implementers must design and use authorization details in a privacy-preserving manner.
+Implementers MUST design and use authorization details in a privacy-preserving manner.
```
Comment (jricher) [inline main.md:953]: We're not supposed to have normative requirements in the privacy section.

```suggestion
It is especially important for implementers to design and use authorization details in a privacy-preserving manner.
```

File: main.md [hunk 3]
```diff
@@ -840,7 +840,7 @@ Figure: Example for authorization details in introspection response.
 
 # Metadata {#metadata}
 
-The AS publishes the list of authorization details types it supports using the metadata parameter `authorization_details_types_supported`, which is a JSON array.
+Authorization servers SHOULD publish the supported list of authorization details types using the metadata parameter `authorization_details_types_supported`, which is a JSON array.
```
Comment (jricher) [inline main.md:843]: Similar to above, if it's included it MUST be in this field.

```suggestion
If the AS wants to advertise its support for this feature, the supported list of authorization details types MUST be included in the AS metadata response [@!RFC8414] using the metadata parameter `authorization_details_types_supported`, which is a JSON array.
```

File: main.md (hunk 2 above)
Comment (b---c) [reply main.md]: There are three other MUST occurrences in Privacy Considerations that should probably be changed too. 

I've never been sure if not supposed to have normative requirements in the privacy section a hard and fast rule. Or recommendation. Or rumor.

Comment (b---c) [reply main.md]: There's a SHOULD in there now too...

Comment (b---c) [reply main.md]: I just went ahead and replaced normative keywords with regular language in the privacy considerations 9d57d65

**Review by jricher — APPROVED** [review]

**Pull request merged**